### ISSUE TITLE
Bug: emojis are not working

### ISSUE THREAD

**Issue by NoodlesDev987** [funkyhippo/modmailbot-formatter#3]
Hello, can you make that emojis will be displayed in logs?

How to get unicode emoji, but idk if also bot can type `\:emoji:` https://github.com/AnIdiotsGuide/discordjs-bot-guide/blob/master/coding-guides/using-emojis.md#but-what-about-unicode-emoji

![image](https://user-images.githubusercontent.com/81709846/126627543-009597d4-1d41-477b-8609-1a88d21a6d97.png)

**Comment by funkyhippo** [comment]
At a cursory glance, I'm not sure if this is trivial since we would have to maintain a mapping from Discord emoji strings to the unicode emoji.

**Comment by NoodlesDev987** [comment]
Yes, also custom emojis should be displayed, but if it's not possible make just that normal emojis will be displayed

**Comment by NoodlesDev987** [comment]
You can use https://github.com/IonicaBizau/emoji-name-map
